### ISSUE TITLE
Unbound local errors for links & links_list vars

### ISSUE THREAD

**Issue by cmac4603** [aboul3la/Sublist3r#119]
Easy fix, I tried to create a branch but didn't have permissions. Putting `links = []` on ln429 and `links_list = []` on 382 fixed these errors I was seeing.

```
Process AskEnum-6:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "sublist3r.py", line 264, in run
    domain_list = self.enumerate()
  File "sublist3r.py", line 237, in enumerate
    links = self.extract_domains(resp)
  File "sublist3r.py", line 395, in extract_domains
    return links_list
UnboundLocalError: local variable 'links_list' referenced before assignment
Process BaiduEnum-2:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "sublist3r.py", line 264, in run
    domain_list = self.enumerate()
  File "sublist3r.py", line 237, in enumerate
    links = self.extract_domains(resp)
  File "sublist3r.py", line 489, in extract_domains
    return links
UnboundLocalError: local variable 'links' referenced before assignment
```

**Comment by lianfeng30** [comment]
i hava the same error

**Comment by ftk-sostupid** [comment]
I got the same error,too.......

**Comment by aboul3la** [comment]
This issue should be fixed now in commit 52089f5761483a0e442229e2c90e317b2e0122fe.

Please pull the new update using command `git pull`
